src/components/ImportForm/__tests__/GitImportForm.spec.tsx

```diff
@@ -66,8 +66,6 @@ describe('GitImportForm', () => {
     routerRenderer(<GitImportForm applicationName={undefined} />);
     expect(screen.getByText('Create application')).toBeDisabled();
     fireEvent.input(screen.getByPlaceholderText('Enter name'), { target: { value: 'test-app' } });
-    expect(screen.getByText('Create application')).not.toBeDisabled();
```

Comment:
Why are you changing a test for import form when there's no change related to import in this PR?

Reply:
removed changes